build_web_compilers/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.2.0
+- Make `librariesPath` configurable in `DevCompilerBuilder`.
```

Comment:
nit: newline above

Reply:
Done

Reply:
heh, I meant between the heading and the bullet but I just went ahead and made the edit :D

Reply:
Ahh, thanks!